### PR TITLE
Update sentiment_analysis.ipynb

### PR DESCRIPTION
**Pull request by Dammy-Joe**
Fixed the broken dataset link and used the iter's next method instead of the next attribute

### CODE REVIEW

File: projects/NaturalLanguageProcessing/sentiment_analysis.ipynb [hunk 1]
```diff
@@ -1,1676 +1,2749 @@
...
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "execution": {},
+        "id": "tyjU0tq3KKQH"
+      },
+      "source": [
+        "# Twitter Sentiment Analysis\n",
+        "\n",
```
Comment (iamzoltan) [inline projects/NaturalLanguageProcessing/sentiment_analysis.ipynb:11]: What happened to the colab and kaggle links?

**Review by iamzoltan — CHANGES_REQUESTED** [review]
It looks like you are using colab and havent cleared your kernel or cells before saving and creating this PR. Can you spin up jupyter notebook locally? Or clear all cells and the kernel before saving? There are too many artifacts contained in the raw data.

**Comment by iamzoltan** [comment]
@Dammy-Joe Please pull the latest changes to hot-fixes branch, and build your changes off that branch. Also please run jupyter locally and clear all cell and kernel before saving

**Comment by Dammy-Joe** [comment]
Thanks @iamzoltan. All the required changes to sentiment_analysis.ipynb have been made in hot_fixes. So, there's no need for a pull request.

**Comment by iamzoltan** [comment]
alright, closing this PR. Let me know if we need to address anything else

**Pull request closed without merging**